### DISCUSSION TITLE
Samples: `temperature` and `top_p`

### DISCUSSION THREAD

**Discussion by garrytrinder** [microsoft/teams-sdk#2220] [Announcements]
In many samples in this repository, prompts are configured with the following:

```json
    "temperature": 0.9,
    "top_p": 0.0,
```

In the [Azure OpenAI On Your Data](https://github.com/microsoft/teams-ai/blob/main/dotnet/samples/08.datasource.azureopenai/) sample for example it uses a high `temperature` and low `top_p`. Is there a specific reason behind this?

https://github.com/microsoft/teams-ai/blob/main/dotnet/samples/08.datasource.azureopenai/Prompts/Chat/config.json

**Comment by heyitsaamir** [comment]
We're clearing discussions and starting fresh. This discussion has not been acted upon. Please reopen the discussion, or start a new one if this question is still important. Thank you for your understanding.
